dynamic_rest/filters.py

```diff
@@ -306,72 +306,60 @@ def rewrite_filters(filters, serializer):
                 q &= ~Q(**{k: v})
         return q
 
-    def _build_prefetch_queryset(
+    def _build_internal_prefetches(
```

Comment:
I wonder if we can be clearer than `internal` vs `external`. Maybe `implicit` (`inferred`?) vs `requested`?